### DISCUSSION TITLE
Date and time visible on the horizon in the dome projection

### DISCUSSION THREAD

**Discussion by PLANETARAS** [Stellarium/stellarium#3518] [Q&A]
Why is it that if in Stellarium 0.23.3 in the config.ini I enter the value "true" in flag_show_tui_datetime the date and time do not appear on the horizon (for dome projection) as it happened with stellarium 0.21.2 ? What's still missing? Why don't you think about inserting the date and time display from the settings dialog? Thank you and clear skies!

**Comment by gzotti** [comment]
Because the whole plugin is missing from 23.3 (we are past the zero, finally!). It will be back in 23.4. 

What do you mean by "inserting the date and time display from the settings dialog"?

**Comment by PLANETARAS** [comment]
Ok thanks for the reply, i'll wait. What does "we are past the zero, finally!" mean?? 
I mean that there is the option to select the display of the date and time on the dome horizon in the settings or options windowsky display.
![Ipotesi spunta Data Ora](https://github.com/Stellarium/stellarium/assets/97767104/0c5644fb-16fd-4cdc-9512-23308d364fc5)
Create a command checkmark like this (example) Display date & time in planetarium fisheye projection. 
Thank you, Stellarium is awesome!! You are very good!!

**Reply by gzotti** [reply]
The TUI plugin is an optional feature used by probably tens of users worldwide. It cannot be configured from the main menu.

You reported something about 0.23.3 which does not exist. "We are past zero" means our version numbers don't any longer start with zero.

**Reply by PLANETARAS** [reply]
Ok thanks, sorry for the oversight on zero
